main.go

```diff
@@ -41,6 +43,16 @@ func main() {
 	// Register the service with the gRPC server
 	snapshotsapi.RegisterSnapshotsServer(rpc, service)
 
+	var gracefulstop = make(chan os.Signal)
+	signal.Notify(gracefulstop, syscall.SIGTERM)
+	signal.Notify(gracefulstop, syscall.SIGINT)
+	signal.Notify(gracefulstop, syscall.SIGSTOP)
+	go func() {
+		<-gracefulstop
+		rpc.GracefulStop()
+		os.Remove(os.Args[1])
+	}()
```

Comment:
Should you call close here?

Reply:
Gracefulstop will call the snapshotter's close automatically. I previously had stop here only to find out that it gets called twice. :)